config/toggles.js

```diff
@@ -51,19 +51,19 @@ function environmentVariableTrue(processEnv, environmentVariableName) {
 const longTermToggles = {
 
   // Disable HTTP, i.e. only use https
-  disableHttp: environmentVariableTrue(pe, 'DISABLE_HTTP'),
+  disableHttp: environmentVariableTrue(pe, 'REQUIRE_HTTPS'),
```

Comment:
I know this isn't specifically in the description (sorry), but I think it'll be easier for everyone in the long run if the toggle attribute names are renamed to requireHttps, enableClockProcess, requireAccessToken, enableWorkerProcess.

(And keep them in alphabetical order here, which will make it easier when there are more of them.)